StandardLibrary/test/TestString.dfy

```diff
@@ -32,4 +34,14 @@ module TestStrings {
     actual := String.HasSubString("large", "larger");
     expect actual == None, "Needle larger than haystack";
   }
-}
+
+  method {:test} TestFileIO()
+  {
+    var x :- expect WriteBytesToFile("MyFile", [1,2,3,4,5]);
+    x :- expect AppendBytesToFile("MyFile", [6,7,8,9,10]);
+    x :- expect AppendBytesToFile("MyFile", [11,12,13,14,15]);
+    var y :- expect ReadBytesFromFile("MyFile");
+    expect y == [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15];
```

Comment:
Since this seems to be the only test of these methods, could you also add

```suggestion
    expect y == [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15];
    x :- expect WriteBytesToFile("MyFile", [1,2,3,4,5]);
    y :- expect ReadBytesFromFile("MyFile");
    expect y == [1,2,3,4,5];
```
?
Just to validate the case where a file already exists for WriteBytesToFile